console/console-init/ui/src/modules/iot-project-detail/components/IoTProjectDetailHeader/IoTProjectDetailHeader.tsx

```diff
@@ -109,8 +109,8 @@ const IoTProjectDetailHeader: React.FunctionComponent<IIoTProjectDetailHeaderPro
       </SplitItem>
     </>
   );
-  const onChange = () => {
-    changeEnable();
+  const onChange = (checked: boolean) => {
```

Comment:
onChangeStatus function name would be more meaningful.

Reply:
Done!